indexer/editable_map_object.cpp

```diff
@@ -83,8 +164,61 @@ void EditableMapObject::SetName(StringUtf8Multilang const & name) { m_name = nam
 void EditableMapObject::SetName(string name, int8_t langCode)
 {
   strings::Trim(name);
-  if (!name.empty())
-    m_name.AddString(langCode, name);
+  if (name.empty())
+    return;
+
+  ASSERT_NOT_EQUAL((int8_t)StringUtf8Multilang::kDefaultCode, langCode,
```

Comment:
1. s/(int8_t)StringUtf8Multilang::kDefaultCode/static_cast<int8_t>(StringUtf8Multilang::kDefaultCode)/
2. Why do you need to cast it at all?